requirements.txt

```
@@ -0,0 +1,14 @@
+click==7.1.2
+dnspython==2.1.0
+email-validator==1.1.2
+Flask==1.1.2
+flask-mongoengine==1.0.0
+Flask-WTF==0.14.3
+idna==3.1
+itsdangerous==1.1.0
+Jinja2==2.11.3
+MarkupSafe==1.1.1
+mongoengine==0.23.0
+pymongo==3.11.3
+Werkzeug==1.0.1
+WTForms==2.3.3
```

Ekstremt mange packages? Du bruker vel ikke mer enn disse?
```suggestion
Flask==1.1.2
Flask-PyMongo==2.3.0
```